modules/new_serverpod_auth/serverpod_auth_jwt/serverpod_auth_jwt_server/test/integration/authentication_tokens_admin_test.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:clock/clock.dart';
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_auth_jwt_server/serverpod_auth_jwt_server.dart';
+import 'package:serverpod_auth_jwt_server/src/generated/refresh_token.dart';
+import 'package:serverpod_auth_user_server/serverpod_auth_user_server.dart';
+import 'package:test/test.dart';
+
+import '../utils/authentication_token_secrets_mock.dart';
+import 'test_tools/serverpod_test_tools.dart';
+
+void main() {
+  withServerpod('Given an auth user with an authentication token,',
+      (final sessionBuilder, final endpoints) {
+    late Session session;
+    late UuidValue authUserId;
+
+    setUpAll(() {
+      AuthenticationTokens.secretsTestOverride =
+          AuthenticationTokenSecretsMock()
+            ..setHs512Algorithm()
+            ..refreshTokenHashPepper = 'pepper123';
+    });
+
+    setUp(() async {
+      session = sessionBuilder.build();
+
+      final authUser = await AuthUser.db.insertRow(
```

Comment:
Need to use `AuthUsers` once https://github.com/serverpod/serverpod/pull/3634 is merged.